Game/Shaders/default_fragment.glsl

```diff
@@ -1,12 +1,11 @@
 #version 460
 
-uniform sampler2D diffuse_texture;
+uniform sampler2D diffuseTexture;
 
 in vec2 uv0;
 
 out vec3 color;
 
-void main()
-{
-	color = texture(diffuse_texture, uv0).xyz;
+void main() {
+	color = texture(diffuseTexture, uv0).xyz;
```

Comment:
Change to .rgb for consistency. Is the same, but for colours is more verbose set .rgb.